tests/console/system_prepare.pm

```diff
@@ -89,7 +89,7 @@ sub run {
             diag "SUSEConnect --status-text locked: $out";
         }
         diag "SUSEConnect --status-text: $out";
-        if (!get_var('MEDIA_UPGRADE')) {
+        if (!get_var('MEDIA_UPGRADE') && !get_var('INSTALL_FOR_MIGRATION')) {
```

Comment:
wouldn't be enough to specify backend spvm or hmc as this only happens with powervm?

Reply:
In fact, this will happened on 1st job for installation of chained jobs for migration test via Full media, so not only power VM but also z/VM, I think we don't need specify the backend, user could set this when needed. I will add comments for when and how to use this setting.